console-subscriber/src/builder.rs

```diff
@@ -101,4 +104,43 @@ impl Builder {
     pub fn build(self) -> (TasksLayer, Server) {
         TasksLayer::build(self)
     }
+
+    /**
+    Configures this builder from a standard set of environment variables:
+
+    * `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+      completed tracing data. Default: 60s.
+    * `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+      localhost:1234 or similar. Default: 127.0.0.1:6669
+    * `TOKIO_CONSOLE_PUBLISH_INTERVAL`: The number of milliseconds to wait
+      between sending updates to the console. Default: 1000ms (1s)
+    */
```

Comment:
style nit: elsewhere we use `///`-style doc comments exclusively...i could be convinced to switch to using block comments for doc comments, but i'd prefer to be consistent about it.

Reply:
I'm a big fan of `/**` / `/*!` for doc comments because it's less visually distracting, but in this case it was just habit and I should have checked for project convention